### ISSUE TITLE
Escape column names 

### ISSUE THREAD

**Issue by stykiaz** [irenical/drowsy#10]
Often columns names will clash with reserved words in MySQl/Postgres/ect .. forcing us to manually escape it and leading to unexpected bugs.

**Comment by tiagosimao** [comment]
True. Do you suggest trying to always escape, allowing some kind of instance-level configuration setting or query-level configuration?
Always escaping should be acceptable if no side effects can occur but since current tests only use PostgreSQL JDBC drivers, we might be missing some odd behaviour here.
What kind of database do you use?

**Comment by stykiaz** [comment]
In our case we are using MySQL/Aurora. I suppose something configurable on instance or a default setting would suffice. A more challenging case would be escaping when column aliases are used or in joins with columns preceded by table aliases.

**Comment by tiagosimao** [comment]
Indeed, or worse, on function calls. Will think for a bit and get back at you.

**Comment by tiagosimao** [comment]
I've added a new method to the SelectBuilder (also available in MySqlSelectBuilder), the identifier() method. It quotes the input with the " character, since it's the SQL standard. I read on the internet (tm) that MySQL and SQL Server have the ` character as default but should also support the double-quote.
If you can, please validate that this method works with your stack and if it does help out.  
I've also added the methods as() and expression() for convenience.
usage example:
```java
SelectBuilder.select().identifier("col1","col2").expression("someFunction()").identifier("col3").as("some_alias")...
```
There are a couple of tests at https://github.com/irenical/drowsy/blob/master/query/src/test/java/org/irenical/drowsy/query/builder/sql/SelectBuilderTest.java  
This should be available on maven central, with the 0.1.7 release in a couple of hours.  
```xml
<dependency>
    <groupId>org.irenical.drowsy</groupId>
    <artifactId>drowsy</artifactId>
    <version>0.1.7</version>
</dependency>
```
